lib/Events/MessageSentEvent.php

```diff
@@ -0,0 +1,92 @@
+<?php declare(strict_types=1);
```

Comment:
I know nextcloud does not follow psr-2 (and not psr-12) but would be nice ;) https://www.php-fig.org/psr/psr-12/#3-declare-statements-namespace-and-import-statements

Reply:
Oh yeah. Didn't know such a guideline existed. I won't change it with this PR but will consider that later, possibly with linter support